### ISSUE TITLE
Will this template pass the review and can be published in chrome? (for new policy)

### ISSUE THREAD

**Issue by comical-interactive** [Kocal/vue-web-extension#413]
This template is so good thank you but i'm wondering will it be rejected when submitting to chrome? did you try it recently? because im worried after take a look at this https://github.com/YuraDev/vue-chrome-extension-template
They said that template is rejected when review because it is obfuscated by webpack. 

Thank you

**Comment by Kocal** [comment]
For Chrome, it seems that they don't really care (or maybe it's because I've uploaded ~15 extensions and two of them have 4k/6k users). :thinking: 
The only one issue I had with them, it was about a bad description for an extension which was used by a lot of people. 
At least put a link to the project and that will be fine: 
![Capture d’écran de 2019-05-12 08-22-20](https://user-images.githubusercontent.com/2103975/57578589-7692fa80-748f-11e9-8e29-1e871a8267a3.png)

When you are publishing your extension for Firefox, you will have a field "comment for the reviewer". Here, you will have to put the link of your project and the build steps. 
Also if your extension use `eval` (in your code or in your vendor), you will have an higher chance to be rejected. This template integrated a script that removes them, but I recently improved this (#398) because the previous behavior worked only with specific version of webpack.

For Opera, you will have to share them a link to the project and give them reproduction steps, here is a screenshot about a moderator:
![Capture d’écran de 2019-05-12 08-19-44](https://user-images.githubusercontent.com/2103975/57578635-5b74ba80-7490-11e9-80de-751550b7284d.png)
